std_msgs/Header header
float64 x_pos              # x_pos reading
float64 x_pos_variance     # x_pos_variance reading
